designs/lily/src/back.mjs

```diff
@@ -552,15 +557,16 @@ export const back = {
     fitKnee: {bool: true, menu: undefined},
     legBalance: 0.5, // between back and front parts
     waistBalance: 0.5,
+    crotchDrop: { pct: 0, min: 0, max: 15, menu: 'advanced' }, // 'downgrade' to advanced menu
     waistHeight: { ...titanBack.options.waistHeight, pct: 50 }, // halfway between waist and hips
     fabricStretch: { pct: 40, min: 0, max: 50, menu: 'fit' },
-    waistEase: {pct: -4,  menu: undefined}, // -fabricStretch/10,
-    seatEase: {pct: -4, menu: undefined}, // -fabricStretch/10,
-    kneeEase: {pct: -4, menu: undefined}, // -fabricStretch/10,
-    test: {pct: ({ options }) => (options.fabricStretch/2)},
+    waistEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
+    seatEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
+    kneeEase: {pct: -4, min: 0, max: 50, menu: 'fit' }, // -fabricStretch/10,
```

Comment:
You have set the default value of `waistEase`, `seatEase`, and `kneeEase` to -4%. However, the minimum value is set to 0%. This seems as if something is incorrect.

Reply:
Indeed, I expect negative values here... now adjusted.